.ci_support/linux_64_.yaml

```diff
@@ -23,3 +29,6 @@ spdlog:
 - '1.11'
 target_platform:
 - linux-64
+zip_keys:
```

Comment:
The numpy pins in `recipe/conda_build_config.yaml` aren't propagated because numpy is not in either the build or host requirements